packages/atomic/src/components/search/atomic-layout/atomic-search-layout.pcss

```diff
@@ -33,6 +38,21 @@ atomic-search-layout {
 
   atomic-layout-section[section='search'] {
     margin: var(--atomic-layout-spacing-y) 0;
+    max-width: var(--atomic-layout-max-search-box-input-width, 678px);
+    width: 100%;
+    justify-self: center;
+
+    /* Only affects desktop */
+    atomic-search-box {
+      &::part(suggestions-double-list) {
+        width: 125%;
+        max-width: var(--atomic-layout-max-search-box-double-suggestions-width, 800px);
```

Comment:
IMO a fixed size would be better, since the objective is probably to ensure we display a certain amount of content without being too large.